src/util-hugepages.c

```diff
@@ -57,15 +68,15 @@ static uint16_t SystemNodeCountGetLinux(void)
     char dir_path[] = "/sys/devices/system/node/";
     DIR *dir = opendir(dir_path);
     if (dir == NULL) {
-        SCLogError("unable to open %s", dir_path);
+        SCLogInfo("unable to open %s", dir_path);
```

Comment:
if we get here, we know the location exists, right? So failure to open now is actually an error?

Reply:
Can be, yeah. I would, however, leave other paths on info level only as those are not gently checked previously (e.g. `/sys/devices/system/node/node%d/hugepages/`). I assume those paths should exist as well but wouldn't want to run into yet another hugepage folder structure and prevent users from running Suricata just because of hugepage analysis. Is it ok?